### DISCUSSION TITLE
Syncing GitHub Issues with GitLab Issues

### DISCUSSION THREAD

**Discussion by ProfessorManhattan** [git-bug/git-bug#787] [General]
I tried connecting both GitLab and GitHub with bridges but I was unable to accomplish what I was looking to do which is sync GL issues with GH issues. Is this possible?

**Comment by MichaelMure** [comment]
At the moment, git-bug forbid this scenario on purpose as this would likely lead to problem. Even if that wasn't the case, you would need credentials for every user on the target bug tracker to replicate the bugs properly, which I assume you don't?

This should be supported some day but will likely require some better story to collect and handle those credentials.
